articles/diagnose-diabetes-with-svm/index.md

```diff
@@ -0,0 +1,319 @@
+- Split Dataset into Training and Test Set
+- Creating the SVM Model
+- Diagnosing a New Patient
+- Assess Model Performance
+
+## EXPLORATORY DATA ANALYSIS WITH PANDAS-PROFILING
+The pandas-profiling library helps us do quick exploratory data analysis with minimal effort. To install pandas-profiling, run the code below:
+
+```python
+pip install pandas-profiling
+```
+
+If you are using Anaconda, then you can run the following code in Anaconda Prompt:
+
+```python
+conda install -c conda-forge pandas-profiling
+```
+
+Now we can import pandas-profiling and generate a report for our dataset. Before we load our dataset, let us import the libraries we will be using.
+
+```python
+# importing libraries
+import numpy as np
+import pandas as pd
+import pandas_profiling
+from sklearn import svm
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import accuracy_score, classification_report
+```
+
+We import our dataset using the read_csv function from pandas. We pass the dataset filename as an argument.
+
+```python
+# Getting Data
+dataset = pd.read_csv("diabetes.csv")
+```
+
+```python
+# generate report with panda-profiling
+profile = dataset.profile_report(title='Diabetes Profiling Report')
+profile
+```
+
+Pandas-profiling gives us the dataset statistics.
+
+![Overview](/engineering-education/diagnose-diabetes-with-svm/1.jpg)
+
+From the report above, we have nine variables and 768 rows. There are no missing data in the dataset. There are no duplicate data.
+
+Pandas-profiling also looks at each of the nine variables. For each variable, it gives us descriptive statistics. It generates a histogram that shows the data distribution of each variable.
+
+![Variables](/engineering-education/diagnose-diabetes-with-svm/2.jpg)
+![Variables](/engineering-education/diagnose-diabetes-with-svm/3.jpg)
+![Variables](/engineering-education/diagnose-diabetes-with-svm/4.jpg)
+![Variables](/engineering-education/diagnose-diabetes-with-svm/5.jpg)
+![Variables](/engineering-education/diagnose-diabetes-with-svm/6.jpg)
+
+We can see the mean, minimum and maximum values of each variable. We can observe the correlation plot between each of the variables. 
+
+![Interactions](/engineering-education/diagnose-diabetes-with-svm/7.jpg)
+
+We can see the Pearson, Spearman, Kendall, and Phik correlation matrix heat map.
+
+![Correlations](/engineering-education/diagnose-diabetes-with-svm/8.jpg)
+
+We can visualize the missing values and know exactly where we have missing values in the dataset. None of the variables contain any missing value, so we can proceed to build our model.
+
+![missing-values](/engineering-education/diagnose-diabetes-with-svm/9.jpg)
+
+Finally, we can view the first ten rows and last ten rows of the dataset.
+
+![first-rows](/engineering-education/diagnose-diabetes-with-svm/10.jpg)
+![last-rows](/engineering-education/diagnose-diabetes-with-svm/11.jpg)
+
+## FEATURE EXTRACTION
```

Comment:
The headings need not be in all uppercase letters.
Please change all the headings as follows:

```markdown
### Feature Extraction
```

Reply:
The headings have been corrected